.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: php
 
 php:
-  - 5.6
+    #  - 5.6
```

Comment:
Have switched to using PHP7 at least for development - guessing that you are going to switch to PHP7 in the near-future anyway?